src/main/webapp/app/exam/participate/exercises/exercise-overview-page/exam-exercise-overview-page.component.html

```diff
@@ -55,12 +55,27 @@ <h2 style="text-align: left; font-weight: normal; margin-bottom: 30px">
                         {{ item.exercise.maxPoints }}
                     </td>
                     <td style="max-width: 50px; font-weight: bold">
-                        <div
-                            style="max-width: 10px"
-                            [ngClass]="setExerciseIconStatus(item)"
-                            [ngbTooltip]="'artemisApp.examParticipation.' + getExerciseButtonTooltip(item.exercise) | artemisTranslate"
-                        >
-                            <fa-icon [icon]="item.icon!" />
+                        <div style="display: flex; align-items: center">
+                            <div
+                                style="max-width: 10px; margin-right: 10px"
+                                [ngClass]="setExerciseIconStatus(item)"
+                                [ngbTooltip]="'artemisApp.examParticipation.' + getExerciseButtonTooltip(item.exercise) | artemisTranslate"
+                            >
+                                <fa-icon [icon]="item.icon!" />
+                            </div>
+                            <div>
+                                @if (item.exercise.studentParticipations !== undefined && item.exercise.studentParticipations.length > 0) {
```

Comment:
We should probably limit this to programming exercises (the only exercise type for which we expect some result during the exam)

![grafik](https://github.com/ls1intum/Artemis/assets/38403547/8d21be47-ce73-439d-999b-0d1de613aac6)